### PR TITLE
(feat): task2 styles

### PR DESCRIPTION
**Pull request by EgidijusPetrauskasVisma**

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
     height: 65px;
     top: 0;
     left: 0;
+    z-index: 4;
```
Comment (zstundys) [inline css/layout.css]: Try to avoid z-indexes if you can -- it's likely this was not needed ;)

File: css/layout.css [hunk 2]
```diff
@@ -14,6 +15,7 @@
     display: flex;
     align-items: center;
     gap: var(--size-2);
+    padding-right: 10px;
```
Comment (zstundys) [inline css/layout.css:18]: This madding is a bit mysterious -- maybe it's not needed?

File: css/layout.css [hunk 3]
```diff
@@ -25,9 +27,8 @@
 }
 
 .footer {
-    height: 80px;
     background-color: var(--primary-light);
-    padding: 0 var(--size-3);
+    padding: var(--size-3) 20px;
```
Comment (zstundys) [inline css/layout.css]: I'd recommend to use a size variable here -- if `--size-3` or `--size-4` does not look right, consider using `calc(var(--size-3) * 1.5)`;

Using size variables tends to result in cleaner looking UIs.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const carousel = document.querySelector('.carousel');
+
+function changeSlide(param) {
+    const currentSlideIndex = Number(getComputedStyle(carousel).getPropertyValue('--slide-index'));
```
Comment (zstundys) [inline script.js]: I'd say the `getComputedStyle` logic here is too much -- I'd recommend simplifying a bit:

```
const ui = {
  carousel: document.querySelector('.carousel'),
  slides: document.querySelectorAll('.slide'),
  slidesCounter: document.querySelector('.slides-counter'),
  slideIndex: 0
}
```

And then just:

```
function nextSlide() {
  if (ui.slideIndex === ui.slides.length - 1) return;

  // rest of the logic
}

function prevSlide() {
  if (ui.slideIndex === 0) return;

  // rest of the logic
}

function updateCount() {
  ui.slidesCounter.textContent = `${ui.slideIndex}/${ui.slides.length}`
}
```

**Review by zstundys — CHANGES_REQUESTED** [review]

File: css/layout.css (hunk 2 above)
Comment (EgidijusPetrauskasVisma) [reply css/layout.css:18]: Without this padding on mobile view nav element align with the side of the screen.

File: css/layout.css (hunk 1 above)
Comment (EgidijusPetrauskasVisma) [reply css/layout.css]: Wihout z-index the carousel element go above header element.

Comment (zstundys) [reply css/layout.css]: Ok, just keep the z-indexes in mind -- it gets way too easy to lose track of them. Personally, I'd keep a list of variables like this:

```
--z-index-header: 1000;
--z-index-dropdown: 2000;
--z-index-tooltip: 3000;
// etc
```

**Review by zstundys — APPROVED** [review]

**Pull request merged**